Doc/whatsnew/3.14.rst

```diff
@@ -75,7 +75,9 @@ New Features
 Other Language Changes
 ======================
 
-
+* :func:`map` now supports :pep:`618`'s ``strict`` parameter, which raises a
+  :exc:`ValueError` if the arguments are exhausted at different lengths.
+  (Contributed by Wannes Boeykens in :gh:`119793`.)
```

Comment:
Is this the correct place to put this?

Reply:
Maybe check where it has been put when `strict` has been added for `zip`.

Reply:
Yes, I correctly guessed it: https://docs.python.org/3.14/whatsnew/3.10.html#other-language-changes
I copied the sentence from there.